scripts/check-examples.sh

```diff
@@ -19,7 +19,7 @@ declare -A results_abi
 
 all_checks_passed=0
 for example in $(ls examples/lang); do
-    cargo build --release --no-default-features --target=wasm32-unknown-unknown --verbose --manifest-path examples/lang/$example/Cargo.toml
+    time cargo build --release --no-default-features --target=wasm32-unknown-unknown --verbose --manifest-path examples/lang/$example/Cargo.toml
```

Comment:
Is the `time` utility pervasive enough to make use of it here? I had to install it first on my machine.

Reply:
removed it